src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let total = 0;
+  const everyDay = 40;
+
+  if (days < 3) {
+    total = days * everyDay;
+  } else if (days >= 3 && days <= 6) {
+    total = days * everyDay - 20;
+  } else if (days >= 7) {
```

Comment:
```suggestion
  } else {
```
Тут то же самое, уже нет смысла проверять >= 7, так как если предыдущие два if не сработали - значит точно либо больше, либо равно 7